ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AUTOVPN_TESTS.yml

```diff
@@ -84,3 +87,51 @@ wan_carriers:
     path_group: INET
   - name: ATT
     path_group: INET
+
+tenants:
+  - name: TenantA
+    vrfs:
+      - name: default
+        vrf_id: 1
+      - name: PROD
+        vrf_id: 42
+      - name: IT
+        vrf_id: 100
+
+wan_virtual_topologies:
+  vrfs:
+    - name: default
+      policy: DEFAULT-AVT-POLICY
+    - name: PROD
+      policy: PROD-AVT-POLICY
+  policies:
+    - name: PROD-AVT-POLICY
+      default_virtual_topology:
+        path_groups:
+          - names: [INET]
+            preference: preferred
+      application_virtual_topologies:
+        - application_profile: voice
+          path_groups:
+            - names: [INET]
+              preference: preferred
+        - application_profile: video
+          path_groups:
+            - names: [INET]
+              preference: preferred
```

Comment:
IMO we should make this the default.

Reply:
done: [3df2218](https://github.com/aristanetworks/ansible-avd/pull/3446/commits/3df22188f41ef26bdd887b1ce5f7d93e99eb3a3a)